server/server.js

```diff
@@ -79,17 +88,26 @@ const server = app.listen(port, () => {
 });
 
 const io = Server.listen(server);
-
 const attachSocketEvents = socketEvents(store);
-
+store.subscribe(
+  () => io.emit('state', store.getState())
+);
 io.on('connection', (socket) => {
-  console.log('connected!');
-  socket.emit('connected', 'hello from the server');
   socket.emit('state', store.getState());
 });
 
 io.on('connection', attachSocketEvents);
-
-store.subscribe(
-  () => io.emit('state', store.getState())
-);
+socketioAuth(io, {
```

Comment:
Not checking subsequent request/action yet. Am thinking it would be done server side where the jwtoken would be verified, and the server would emit back the response, or unauthorized.